src/test/ui/proc-macro/parent-source-spans.stderr

```diff
@@ -132,6 +132,11 @@ LL |     parent_source_spans!($($tokens)*);
 ...
 LL |     one!("hello", "world");
    |     ----------------------- in this macro invocation
+   | 
+  ::: $SRC_DIR/libcore/result.rs:LL:COL
+   |
+LL |     Ok(#[stable(feature = "rust1", since = "1.0.0")] T),
+   |     --------------------------------------------------- similarly named tuple variant `Ok` defined here
```

Comment:
Somewhat unexpectedly (but not unforeseeably), this change now makes more obvious that attribute annotations are part of the public documentation and will be more visible than currently. In some cases like this one it might make sense to tweak them "out of the way" to improve readability while keeping the same semantic meaning.